assets/libs/hamburgers-master/dist/hamburgers.css

```diff
@@ -17,7 +17,7 @@
   text-transform: none;
   background-color: transparent;
   border: 0;
-  margin: 0;
+  margin: 0;                     
```

Comment:
Zmiany z całego pliku mają zostać przywrócone do wersji pierwotnej. Można to zrobić za pomocą gita, ale nie chcę Ci namieszać. Weź po prostu ręcznie wywal ten plik i ręcznie wrzuć oryginał.

Reply:
Nie tyczy się to tylko tej linijki

Reply:
Poprawiłem